dev/release/verify-release-candidate.sh

```diff
@@ -128,7 +128,6 @@ test_source_distribution() {
   python3 -m venv .venv
   source .venv/bin/activate
   python3 -m pip install -U pip
-  python3 -m pip install -r requirements-310.txt
```

Comment:
I need to add `python3 -m pip install -U maturin` to make the verify script work locally.

Reply:
that makes sense, we create and activate a new venv right above

Looks like `requirements-310.txt` was removed in #994